src/main/java/io/aiven/kafka/auth/json/AivenAcl.java

```diff
@@ -70,9 +78,7 @@ public Boolean check(final String principalType,
                     final Pattern resourceRe = Pattern.compile(resourceReStr);
                     mr = resourceRe.matcher(resource);
                 }
-                if (mr != null && mr.find()) {
-                    return true;
-                }
+                return mr != null && mr.find();
```

Comment:
Not that I don't agree with pretty sensible small changes like this (and maybe in some other places), I would keep them out of this PR to reduce the review surface.